src/Aspire.Hosting/Dcp/DcpHost.cs

```diff
@@ -366,4 +370,120 @@ void LogLines(in ReadOnlySequence<byte> buffer, out SequencePosition position)
             reader.Complete();
         }
     }
+
+    private void TryShowContainerRuntimeNotification(DcpInfo dcpInfo, CancellationToken cancellationToken)
+    {
+        // Check if the interaction service is available (dashboard enabled)
+        if (!_interactionService.IsAvailable)
+        {
+            return;
+        }
+
+        var containerRuntime = _dcpOptions.ContainerRuntime;
+        if (string.IsNullOrEmpty(containerRuntime))
+        {
+            // Default runtime is Docker
+            containerRuntime = "docker";
+        }
+
+        var installed = dcpInfo.Containers?.Installed ?? false;
+        var running = dcpInfo.Containers?.Running ?? false;
+
+        // Only show notification if container runtime is installed but not running
```

Comment:
If you don't have a container runtime installed at all and your resource model containers container resources, then we could display a message to install the container runtime isntead>

Reply:
OK doing a follow up here https://github.com/dotnet/aspire/pull/11041